src/main/java/duke/Dishes/DishList.java

```diff
@@ -0,0 +1,36 @@
+package duke.Dishes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DishList {
+    private List<Dishes> dishlist;
```

Comment:
Important that you write proper JavaDoc comments for all classes. You can find the Java Coding Standards on the module website